src/Api/User/saveRecipe/index.js

```diff
@@ -1,19 +1,18 @@
-import { getUser, useRemoveToken } from '../..';
+import { getUserId, useRemoveToken } from '../..';
 import axios from '../../../axiosConfig'
 import { useCookies } from 'react-cookie';
 import { enqueueSnackbar } from 'notistack';
 
 const useSaveRecipe = () => {
-  let user = getUser();
+  let userId = getUserId();
   const { checkIfInvalidToken } = useRemoveToken();
   const [cookies, setCookies] = useCookies(['access_token']);
   const axiosSaveRecipe = async (recipeId) => {
     try {
-      const res = await axios.put('/recipes/saveRecipe', { recipeId, userID:user._id }, { headers: { authorization:cookies.access_token } });
+      const res = await axios.put('/recipes/saveRecipe', { recipeId, userId }, { headers: { authorization:cookies.access_token } });
```

Comment:
Extract http request to a separate file/ custom hook if needed

Reply:
But it's already inside a custom hook